egs/public_dataset/jsut-song_ver1/conf/sing_quality.csv

```diff
@@ -0,0 +1,552 @@
+id,quality

```

Comment:
I think we won't need this for jsut-song.

The quality measure mainly works for our self-collected datasets with different SNR